modules/raft/src/main/java/org/apache/ignite/raft/jraft/core/FSMCallerImpl.java

```diff
@@ -665,6 +676,27 @@ private void doSnapshotLoad(final LoadSnapshotClosure done) {
             setError(e);
             return;
         }
+
+        // JRaft tests use such strange metas, so we have to protect... in production, these are never null.
+        if (meta.peersList() != null && meta.learnersList() != null) {
```

Comment:
I clarified the comment. I made the condition the most protective just in case as here we are protecting from test code. In production, this doesn't matter, neither peers no learners can be null.